graphql/nftLoans/queries/eventsForAddress.graphql

```diff
@@ -13,7 +13,7 @@ query createAndClose(
   ) {
     ...createEventProperties
     loan {
-      id
+      ...allLoanProperties
```

Comment:
graphQL ftw, we include the associated loan in the event we query for and then it Just Works:tm: